### ISSUE TITLE
ឞ 179e + ា 17B6 ligature is the wrong shape

### ISSUE THREAD

**Issue by livingfield** [silnrsi/font-busra#2]
All Khmer fonts form a ligature ប 1794 + ា 17b6 = បា. The ា 17b6 vowel normally attaches to the nearest point of the preceding consonant. If it were to attach in this way to ប 1794, however, the resulting shape would be easily confused with ហ 17a0. All Khmer fonts, therefore, extend the hook at the left across the top of the ប to attach to ា.
The Khmer letter ឞ 179e is an archaic letter which is never used in Modern Khmer. Its only use today is in the orthographies of several minority languages, where it symbolizes a sound similar to ប 1794. Students never see this letter in Khmer school. Readers learn it as "ប with an extra stroke”. 
There is no standard appearance for ឞ 179e + ា. Existing Khmer fonts are split pretty evenly between the two obvious choices:
(1) Follow the pattern for ប + ា. Stretch the hook on the left over the top of the consonant in order to attach it to the ា. Readers consider ឞ 179e to be a variant of ប 1794 so they expect the ligature to follow the same pattern. NGO staff teaching literacy in Ratanakiri prefer this shape.
(2) Since ឞ 179e has an extra stroke on the right vertical stroke, it should not be confused with ហ 17a0. Therefore it is unnecessary to extend the hook from the left over the top of the consonant. Experience has shown, however, that new readers do struggle to read the ligature without the extension over the top. They think it’s a "ហ 17a0 with a hook”, following in the pattern of ប/ឞ and គ/ឝ. When the Khmer Mondulkiri font family was designed, someone from the Ministry of Education said that this was the correct form, but other officials have since contradicted this. In any case, it seems unreasonable to expect Khmer to have a “correct” form for a letter combination which simply does not exist at all in Khmer.
<img width="118" alt="SSO-AA" src="https://github.com/silnrsi/font-busra/assets/15900199/ff80a1ea-e3d9-42b2-b22e-d42fe9e7e6db">

**Comment by viverechristus** [comment]
The translators on the team I work with have often complained about the "Bad" version in which  ា 17b6 connects to right ascending arm of ឞ 179e rather than passing over a lowered right arm to connect to the left ascending arm, as in បា (the "Good" version). Their experience with readers in their own villages has also shown a strong community preference for the "Good" rendering as identified in @livingfield's remark above.

**Comment by mhosken** [comment]
![Image](https://github.com/user-attachments/assets/a8f13004-9b61-4650-81dc-1b8d9679fd44)
The first shows U+1794 U+17B6 and the second, 179E 17B6.
Does this mean it's fixed?

**Comment by viverechristus** [comment]
I'm afraid that's incorrect.

**Comment by viverechristus** [comment]
Just to clarify, the behavior of the two should be identical, with 17B6
connecting to 179E on the left and crossing over the body of 179E, as it
does with 1794

**Comment by mhosken** [comment]
Are you saying that 1794+17B6 should look identical to 179E+17B6? If so, that would violate the principle that there should not be 2 ways to encode the same valid visual representation. What needs to happen to make one of them look different?

**Comment by livingfield** [comment]
> Are you saying that 1794+17B6 should look identical to 179E+17B6?
No, 1794+17B6 is OK as it is, but 179E+17B6 should look like the green "good" picture above. It should be identical to 1794+17B6 with an added stroke at the center.

**Comment by bhirsb** [comment]
New 179E+17B6 (right), next to 1794+17BC (left). Is this correct? 

<img width="830" alt="Image" src="https://github.com/user-attachments/assets/0ffbfa27-5de1-4ac7-8776-9ef6c5152f3c" />

**Comment by viverechristus** [comment]
> New 179E+17B6 (right), next to 1794+17BC (left). Is this correct?

Yes, this is correct. Thanks!
